lib/yast2_vnc_shortcuts.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Shortcuts to configure remote administration with yast2 vnc
+# Maintainer: Joaquín Rivera <jeriveramoya@suse.com>
+
+package yast2_vnc_shortcuts;
+
+use base Exporter;
+use Exporter;
+
+use strict;
+use testapi;
+use version_utils qw(is_leap is_sle is_tumbleweed);
+
+our @EXPORT = qw($is_newer_product %remote_admin %firewall_settings %firewall_details $confirm);
+
+our $is_newer_product = (is_sle('>=15') || is_leap('>=15.0') || is_tumbleweed);
```

Comment:
again, as always, I propose to turn it around and never specify tumbleweed explicitly but consider it the default. You can still specify "is_newer_product" if you like by negating the whole check

Reply:
It's actually easier: `is_sle('>=15') || (is_opensuse && !is_leap('15.0'))`
I find it more readable than negation of statement with multiple AND and OR conditions. It's same as `unless`, some kind of not natural =)